### DISCUSSION TITLE
Replay is getting stuck after pausing and cannot navigate anymore

### DISCUSSION THREAD

**Discussion by nagatsuka-shuuya** [ppy/osu#25147] [Q&A] [answered]
### Type

Game behaviour

### Bug description

1. Usually I navigate throw the replay using keys (Arrow Left, Arrow Right, Space). At some point after pressing pause replay I cannot navigate through the replay anymore even pressing Space to resume the replay. The logs are attached.
Actually for this one I cannot reproduce it anymore, so If there is nothing suspicious in logs please close the issue.

2. Is it expected I cannot navigate forward or backward after pressing Pause? So I need to resume the replay first and then use arrows to go forward or backward?

3. Also I faced another issue (unfortunately I don't have logs and I cannot reproduce it anymore, so just mention it here).
At some point when I navigated to the replay beginning the music has gone (while the hitsounds worked properly), so the replay was with hitsounds only. But If I navigated forward, then the music has appeared, and after moving back to the replay beginning the music has been gone again.

### Version

2023.1008.1

### Logs

[database.log](https://github.com/ppy/osu/files/12918838/database.log)
[input.log](https://github.com/ppy/osu/files/12918839/input.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/12918840/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/12918841/network.log)
[performance.log](https://github.com/ppy/osu/files/12918842/performance.log)
[performance-audio.log](https://github.com/ppy/osu/files/12918843/performance-audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/12918844/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/12918845/performance-input.log)
[performance-update.log](https://github.com/ppy/osu/files/12918846/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/12918847/runtime.log)
[updater.log](https://github.com/ppy/osu/files/12918849/updater.log)

**Comment by bdach** [comment, marked as answer]
Please don't quadruple post, this is not a chatroom.

Without a video I'm not sure anything can be done here. There is stuff in logs but I'm not even sure which stuff I should be looking at specifically. As such, converting to discussion for now, until concrete reproduction steps or additional evidence is provided.

**Reply by nagatsuka-shuuya** [reply]
As always thank you for checking.
1. Moved all issues to the description block.
2. For items 1 and 3 in the description as I said I cannot reproduce it anymore so please feel free to close as "Can't reproduce", reported just in case you have some internal understanding of reasons It happened.
3. For the item 2 in the desc, this is mostly a question, so we can keep a discussion for it.

**Reply by bdach** [reply]
Second question would probably be in scope of https://github.com/ppy/osu/issues/25146 I think?

**Reply by nagatsuka-shuuya** [reply]
In my opinion essentially It's the same origin, but different issues:
1. It's not-intuitive you need to press Resume first to be able navigate fast to some replay time point (actually I personally was confused when I tried to quickly go to the replay end, but I couldn't since It was paused, maybe It's just me XD) <- This is not really an issue since you still can press resume first and then navigate with arrows, but I think It can be improved, so I asked here.
2. When I currently see the 30sec paused replay and I press the Resume button I expect to see the replay from the same 30 sec, but actually It fast scrolling to another time point and It looks really bad. By accident It's also due to previously pressed arrows, but actually an another issue.

To summurise, I am probably ok with the 1st, but not ok with the 2nd.

Anyways I also want to keep tracking it in one place, so I suggest to close this discussion here since It's mostly about the case when I pressed all buttons (even Space to resume), but the replay was stuck so I needed to escape it.
